e2e-tests/setup/setup.ts

```diff
@@ -32,7 +32,7 @@ export default async () => {
 		{
 			title: 'Create Directus Docker Image',
 			task: async (_, task) => {
-				const result = await globby(['**/*', '!node_modules', '!**/node_modules', '!**/src'], {
+				const result = await globby(['**/*', '!node_modules', '!**/node_modules'], {
```

Comment:
This is by design, so we don't copy over code that isn't needed to run Directus (assuming you run `npm run build` outside of the image)